Create SECURITY.md (#973)
jpadilla/pyjwt: SECURITY.md

# Security Policy

## Supported Versions

The following versions of this project are currently being supported with security updates.

| Version | Supported          |
| ------- | ------------------ |
| 2.9.x   | :white_check_mark: |
| < 2.8   | :x:                |

## Reporting a Vulnerability

> Please reach out to auvipy@gmail.com for reporting security concerns via email.

Reviewer: ```suggestion
In order for the vulnerability reports to reach maintainers as soon as possible, the preferred way is to use the "Report a vulnerability" button under the "Security" tab of the associated GitHub project. This creates a private communication channel between the reporter and the maintainers.

If you are absolutely unable to or have strong reasons not to use GitHub's vulnerability reporting workflow, please reach out to [security@jpadilla.com](mailto:security@jpadilla.com).
```

# Security Policy

## Supported Versions

> This section tell people about which versions of this project are
> currently being supported with security updates.

Reviewer: ```suggestion
The following versions of this project are currently being supported with security updates.
```
